modules/ROOT/pages/create-instance-task-flex.adoc

```diff
@@ -24,18 +24,141 @@ See xref:gateway::flex-install.adoc[Install Flex Gateway] and xref:gateway::flex
++
+Host::::
++
+The *Host* rule defines the request host the route can service.
++
+You can only define one host URL for each route. Only requests made from the defined host are sent to this route.
++
+To define the *Host* rule:
++
+. Enter your host in the *Host* configuration field.
++
+Header::::
++
+The *Headers* rule defines what headers are required to be present or not present for this route to service the request.
++
+For this rule, you must define the header name and a `true` or `false` value of whether or not the header is present. Only requests that meet all the specified header requirements are sent to this route. Additional headers present in the request that are not specifically defined in the rules are ignored. 
++
+You can define up to 10 headers.
++
+To define the *Headers* rule:
++
+. Click *Add header (optional)*, if this is not your first header.
+. Enter the header name in the first box.
+. Enter `true` or `false` dependent on if the header must be present.
++
+NOTE: To delete a header. click the *Trash Can* icon (image:trash-can-icon.png[2%,2%]).  
+
+==== Route Order
+
+In addition to route rules, you can direct requests to different routes by defining the route order. 
+
+Flex Gateway directs requests to the first route where the request meets the route rules. Therefore, route ordering is very important when a request may meet the route rules of multiple routes. 
+
+For example, in a configuration where route one has the `GET` method defined as a rule and route two has no route rules defined, all `GET` requests are sent to route one and all other requests are sent to route two. If the route order was reversed and route one had no route rules, Flex Gateway would direct all requests to route one before any `GET` requests could reach route two.
+ 
+Generally, you should order the routes with more complex route rules first. If you do not define a route order, routes are ordered in the order they were created.
+
+To reorder routes:
+
+. Click *Reorder Routes*.
+. Use the up and down arrows to rearrange the route order.
+. Click *Apply Order*
+
+=== Upstreams
+
+You can use multiple upstream services in a single route to direct requests to similar services. For example, if you want to test the performance of a new beta upstream service without sending all traffic to the new service, you can direct half of the traffic to a stable upstream service and half to the new upstream service.
+
+Each API instance route can support up to 10 upstream services, but at least one upstream service is required for each route. 
+
+What upstream service within the route each request is sent to is random and independent of any previous request. The upstream *Weight* defines the percentage chance of a request being sent to a particular upstream service.
+
+Since any upstream service within a route could recieve any request, all upstream services within the same route must adhere to the same API contract.
```

Comment:
```suggestion
Because any upstream service within a route can receive any request, all upstream services within the same route must adhere to the same API contract.
```